.github/workflows/beeler-hw.yml

```diff
@@ -0,0 +1,39 @@
+name: GitHub Actions Homework
+run-name: Hello World Workflow
+on: [push]
+jobs:
+  Run-Hello-World:
+    name: Built and run on ${{ matrix.os }} with ${{ matrix.compiler }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+        compiler: [g++-10]
+        include:
+          - os: ubuntu-latest
+            compiler: g++-10
+
+          - os: macos-latest
+            compiler: g++-10
+
+          - os: windows-latest
```

Comment:
oof talk about punishing yourself.

Reply:
Oh my god, I thought the homework requirement was "both windows and mac," not linux. I'm crying

Reply:
haha I'd never as you to suffer that much